src/celeritas/ext/GeantImporter.cc

```diff
@@ -147,6 +153,8 @@ struct ProcessFilter
         {
             case G4ProcessType::fElectromagnetic:
                 return (which & DataSelection::em);
+            case G4ProcessType::fOptical:
```

Comment:
I still have to go through most of the PR, but fun fact: scintillation is tagged as `fElectromagnetic` in Geant4. So we'll have to be careful with our physics flags if we plan on saying that scintillation (though physically correct to be an EM process) belongs to optical, as it emits optical photons.

Reply:
Yeesh. Thanks for the heads up. Guess that's because it's affecting EM tracks and not optical photons??

Reply:
Probably? That would be my first guess too... Also, for scintillation you need ionization on (you need dE/dx), so that may be another reason?